### ISSUE TITLE
Add gif of Ozone being used to README. Would help first time users understand how to use the code.

### ISSUE THREAD

**Issue by Milind220** [Milind220/Ozone#14]
Make a gif of a screen recording of yourself using the package, and add it to the README. Make sure to try both the currently available methods in Ozone to showcase it's capabilities properly.

**Comment by tariksouabny** [comment]
Are the two methods `get_city_air()` and `get_multiple_city_air`?

**Comment by Milind220** [comment]
Yeah, that' correct! The docstrings for the methods have the info on what arguments to use with them too. The README of the repo also has the info

**Comment by tariksouabny** [comment]
Also, where do you want the gifs? Under what section? Is it at the top?

**Comment by Milind220** [comment]
@tariksouabny I think the 'getting started' section of the README is probably the best place. Just under the code snippets that are there right now.

**Comment by tariksouabny** [comment]
Finished adding the Gifs in. Now they are ready for review!

Thank you!

**Comment by Milind220** [comment]
@tariksouabny I'll review that right away! Thanks a lot for your contribution 💯
